text/profiles/0212-profiling-vision.md

```diff
@@ -0,0 +1,96 @@
+# Propose OpenTelemetry Profiling Vision
```

Comment:
Editorial: you may want to hard wrap the lines to some reasonable number f characters (e.g. 80 or 100) to make reviewing easier. Otherwise comments are attached to entire paragraphs instead of targeting individual lines where necessary.